src/plugins/custom_import_map/public/components/vector_upload_options.test.tsx

```diff
@@ -209,6 +211,8 @@ describe('vector_upload_options', () => {
     await waitFor(() => {
       fireEvent.click(button);
     });
+    const customerCalloutDiv = screen.getByTestId('customerCallout');
+    expect(customerCalloutDiv).toBeTruthy();
```

Comment:
what will happen if this is not Truthy?

Reply:
This is just checking presence of the div in the success and partial success scenarios. So not Truthy would mean that we are not getting the experience as intended i.e the callout is not part of the UI in case of success and partial success scenarios.